Dockerfile

```diff
@@ -26,7 +26,7 @@ COPY root /
 
 RUN apt-get update \
     && ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone \
-    && apt-get install --no-install-recommends -y make rsync openjdk-11-jdk-headless curl graphviz ruby fontconfig graphviz jq \
+    && apt-get install --no-install-recommends -y make rsync openjdk-11-jdk-headless curl graphviz ruby fontconfig graphviz jq git \
```

Comment:
Please add a simple test that check if git is installed like

https://github.com/adorsys-containers/arc42-tools/blob/44dd903a870c35472aaf8e24ff3f891c8075adaf/tests/tests.sh#L7